### ISSUE TITLE
Epoch Training Loss Validation Loss

### ISSUE THREAD

**Issue by 12mohaned** [aub-mind/Arabic-Empathetic-Chatbot#1]
First, I would like to thank you for the dataset. It did help me a lot in my thesis, good work. Second, when I run the model on Colab it crashes with output  `Your session crashed after using all available RAM `.

I would like to know if the model requires specific computational specs since it is not mentioned.

Best Regards.

**Comment by tareknaous** [comment]
@12mohaned we have uploaded the pre-trained model to huggingface for ease-of-use. Please check the updated repo!

**Comment by Munirah-kh** [comment]
can u please tell mw the steps to run the code on colab ?

**Comment by Munirah-kh** [comment]
can u please tell me the steps to run the code on colab ?
